### PR TITLE
Added port setting

### PR DESCRIPTION
**Pull request by alallier**
Add port setting to allow user defined control of the port the server runs on

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,11 @@
           "type": "boolean",
           "default": true,
           "description": "Automatically open the browser window when opening or editting a markdown file"
+        },
+        "instantmarkdown.port": {
+          "type": "number",
+          "default": 8090,
+          "description": "Port to run the server on"
```
Comment (alallier) [inline package.json]: Let me know if you want a wording change here.

**Comment by dbankier** [comment]
Sounds good. The PR now conflicts (since we didn't ignore the out build). 
Will merge as soon as the conflict is fixed.
Down the track, I would like to use a port range instead. It will close some other issues.

**Comment by alallier** [comment]
Resolved conflicts. I'm new to VS Code extensions, should the out directory be ignored?

**Comment by alallier** [comment]
Any updates on taking this and publishing the update for my [other merge request](#32)? (Or at least publishing for #32)?

**Comment by dbankier** [comment]
Sorry about that. Let this one slip. Will publish now.

**Comment by alallier** [comment]
That' alright. Thank you for accepting and publishing!

**Pull request merged**